core/lib/api.ts

```diff
@@ -7,3 +7,18 @@ import { env } from "./env/env";
 export const client = initTsrReactQuery(siteApi, {
 	baseUrl: typeof window === "undefined" ? env.PUBPUB_URL : window.location.origin,
 });
+
+export const RETRY_COUNT = 3;
+
+export const retryPolicy = (failureCount: number, error: Error) => {
+	if (!("status" in error) || typeof error.status !== "number") {
+		return failureCount < RETRY_COUNT;
+	}
+
+	if (error.status >= 400 && error.status < 500) {
+		// unlikely such an error would resolve itself, so don't retry
+		return false;
+	}
+
+	return failureCount < RETRY_COUNT;
+};
```

Comment:
By default ReactQuery retries a query up to three times if it fails, but the new forms route throws a 400 if the thing is not working properly and therefore it will retry it a bunch of times. In general, I don't think it makes sense to retry on a 400 or higher, so i added this custom retry policy